sound/ai/syndicate/script.txt

```diff
@@ -0,0 +1,38 @@
+aimalf.ogg 		artifical core has been subverted
+aliens.ogg 		alien lifeforms have been detected
+anime.ogg 		anime has been detected retreat
+granomalies.ogg		generation of gravitational anomalies has been successful
+intercept.ogg 		nano tray sen command has detected our agents aboard the station
+ionstorm.ogg 		artificial core minor subversion successful
+meteors.ogg 		watch for flying rocks
+outbreak5.ogg 		all nuclear ops your mission is cancelled return to base
+outbreak7.ogg 		viral culture has been implanted on ship
+poweroff.ogg 		implanted corrupt code in lighting subroutine
+poweron.ogg 		nano tray sen command is smarter than we thought
+radiation.ogg 		radioactive frequencies have been broadcasted to the station
+shuttlecalled.ogg 	our work here is done
+shuttledock.ogg 	your precious emergency shuttle is here get on
```

Comment:
this one is weird idk

Reply:
which one

Reply:
your precious emergency shuttle is here get on